examples/test/test_bert_ptq_cpu.py

```diff
@@ -23,12 +23,15 @@ def setup():
 @pytest.mark.parametrize("search_algorithm", ["tpe"])
 @pytest.mark.parametrize("execution_order", ["joint"])
 @pytest.mark.parametrize("system", ["local_system", "aml_system", "docker_system"])
-@pytest.mark.parametrize("olive_json", ["bert_ptq_cpu.json"])
+@pytest.mark.parametrize("olive_json", ["bert_ptq_cpu.json", "bert_inc_ptq_cpu.json", "bert_inc_smoothquant_ptq_cpu.json"])
```

Comment:
Emm, if so, current Bert pipeline will run with about 3x times than before(about 1h). As it is only test the inc quantizaion passes, maybe we do not need to run the complete E2E tests?

I believe INC team already had their tests but not in Olive just like other ONNX passes.

Reply:
Thanks for your suggestion. I removed Bert INC test from the CI pipeline. All INC-related examples are tested internally by INC team.